src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php

```diff
@@ -194,4 +220,55 @@ private function normalizeFunctionNames(array $functionNames)
             return \strtolower($functionName);
         }, $functionNames);
     }
+
+    /**
+     * @return string[]
+     */
+    private function getDefinedFunctions()
+    {
+        if (!$this->configuration['opcache-only']) {
+            return \get_defined_functions()['internal'];
+        }
+
+        return [
+            'array_slice',
+            'assert',
+            'boolval',
+            'call_user_func',
+            'call_user_func_array',
+            'chr',
+            'count',
+            'defined',
+            'doubleval',
+            'floatval',
+            'func_get_args',
+            'func_num_args',
+            'get_called_class',
+            'get_class',
+            'gettype',
+            'in_array',
+            'intval',
+            'is_array',
+            'is_bool',
+            'is_double',
+            'is_float',
+            'is_int',
+            'is_integer',
+            'is_long',
+            'is_null',
+            'is_object',
+            'is_real',
+            'is_resource',
+            'is_string',
+            'ord',
+            'strlen',
+            'strval',
+            'function_exists',
+            'is_callable',
+            'extension_loaded',
+            'dirname',
+            'constant',
+            'define',
```

Comment:
What do you think about keeping items in this list sorted - this makes it easier to a) find and b) decide where to add items?

Reply:
Changed in 86c225c63c7ee80ee399574c6b4a895f2329f14a.